packages/devtools_app/lib/src/shared/common_widgets.dart

```diff
@@ -295,34 +295,17 @@ class SettingsOutlinedButton extends GaDevToolsButton {
   }) : super(outlined: true, icon: Icons.settings_outlined);
 }
 
-class HelpButton extends StatelessWidget {
```

Comment:
the changes related to help buttons and dialogs were broken out into https://github.com/flutter/devtools/pull/7138. Please review these changes there. Will land that PR and rebase once approved and green.